firestore.rules

```diff
@@ -24,7 +24,7 @@ service cloud.firestore {
     match /deals-token-swap/{deal} {
       allow read: if isPublicDeal() || userIsProposalLead() || userIsRepresentative();
       allow update: if userIsProposalLead() || (userIsAuthenticated() && updateOnlyToDiscussions());
-      allow create, delete: if false;
+      allow create, delete: if true;
```

Comment:
Does this mean, anyone will be able to create?

Reply:
I wanted to ask to @BartuCulha about this.  Not sure we really care about this rule in anything other than prod.

Reply:
cc @paweljanicki